core/kazoo_services/src/kz_services.erl

```diff
@@ -673,13 +696,19 @@ is_good_standing(Thing) ->
 %% * All other cases the account is not in good standing
 %% @end
 %%------------------------------------------------------------------------------
+%% @equiv is_good_standing(Thing, 0)
```

Comment:
This is probably breaking edoc. `equiv` is only shown for this function and all other comments are ignored. And since you moved the comments for `is_good_standing/2` here, eodc is not generating any doc for arity 2.

Reply:
@icehess I will fix this on master, but accept here to allow the build to continue.